### PR TITLE
Rename "addons" to "marketplace" on the UI and URL

### LINKED ISSUE #2616: Rename references from "add-on" to "Marketplace"
- Every reference to "add-ons" should be replaced to "Marketplace", as per discussed on Slack: p1697138450380629-slack-C036R3Y5ET0
- This is a continuation to #2586

### PR DESCRIPTION
**Pull request by fjorgemota**
Fixes #2616

### Changes Proposed in this Pull Request

* Change every URL and UI reference from "add-ons"  to "marketplace" (or "extensions", where appropriate)
  * This change doesn't change any filenames, classes, method, functions or hook names

### Testing Instructions

Navigate WPJM and make sure all links to the previous addons page still works correctly. Check notices and various possible references to the pages.

### Release Notes

- Use "extensions" instead of "add-ons" on the URLs and UI

### Screenshot / Video

----

| Plugin build for 67a7b6f8d1d96474704eaff86f6e819e50c38bf0 <a href="#"></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/10/wp-job-manager-zip-2622-67a7b6f8.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/10/2622-67a7b6f8)             |

### CODE REVIEW

File: includes/admin/views/html-admin-page-addons.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	exit;
 }
 
-echo '<h1 class="screen-reader-text">' . esc_html__( 'WP Job Manager Add-ons', 'wp-job-manager' ) . '</h1>';
+echo '<h1 class="screen-reader-text">' . esc_html__( 'WP Job Manager Marketplace', 'wp-job-manager' ) . '</h1>';
```
Comment (yscik) [inline includes/admin/views/html-admin-page-addons.php:12]: Should we simply this to "Marketplace", or maybe "Job Manager Marketplace"?

image

File: readme.txt [hunk 2]
```diff
@@ -57,9 +57,9 @@ Resume Manager is a plugin built on top of WP Job Manager which adds a resume su
 
 Allow registered users to save their job searches and create alerts which send new jobs via email daily, weekly or fortnightly.
 
-**[Core add-on bundle](https://wpjobmanager.com/add-ons/bundle/)**
+**[WPJM Pro Bundle](https://wpjobmanager.com/add-ons/bundle/)**
```
Comment (yscik) [inline readme.txt]: How does this sound?
```suggestion
**[Job Manager Pro Bundle](https://wpjobmanager.com/add-ons/bundle/)**
```

**Review by yscik — APPROVED** [review]
Looking good!

A worry I thought of is will people find the "Licenses" page under "Marketplace". But we do link to it directly so it's probably fine.

File: includes/admin/views/html-admin-page-addons.php (hunk 1 above)
Comment (mikeyarce) [reply includes/admin/views/html-admin-page-addons.php:12]: My vote is for "Job Manager Marketplace".

Comment (thedebian) [reply includes/admin/views/html-admin-page-addons.php:12]: Just "Marketplace" 👨‍🍳 💋

Comment (fjorgemota) [reply includes/admin/views/html-admin-page-addons.php:12]: Renamed to "Marketplace" in aaff8c2. Thanks!

File: readme.txt (hunk 2 above)
Comment (fjorgemota) [reply readme.txt]: Sounds good to me! Renamed in 9e45a14. Thanks!

**Comment by fjorgemota** [comment]
@yscik Thanks for the review and the approval! I sent it for review again mainly because I fixed a bunch of conflicts and made the requested changes, so I'd appreciate an extra look. Thanks!

File: readme.txt [hunk 3]
```diff
@@ -31,15 +31,15 @@ The plugin comes with several shortcodes to output jobs in various formats, and
...
-= Add-ons =
+= Extensions =
 
 The core WP Job Manager plugin is free and always will be. It covers all functionality we consider 'core' to running a simple job board site.
 
-Additional, advanced functionality is available through add-ons. Not only do these extend the usefulness of the core plugin, they also help fund the development and support of core.
+Additional, advanced functionality is available through extensions. Not only do these extend the usefulness of the core plugin, they also help fund the development and support of core.
 
-You can browse available add-ons after installing the plugin by going to `Job Listings > Add-ons`. Our popular add-ons include:
+You can browse available extensions after installing the plugin by going to `WP Job Manager > Marketplace`. Our popular extensions include:
```
Comment (yscik) [inline readme.txt]: One last tiny thing, this should be the main menu name: image

```suggestion
You can browse available extensions after installing the plugin by going to `Job Manager > Marketplace`. Our popular extensions include:
```

**Review by yscik — APPROVED** [review]
Looks good!

File: readme.txt (hunk 3 above)
Comment (fjorgemota) [reply readme.txt]: Great catch! Fixed in 67a7b6f.  Thanks!

**Pull request merged**